app/components/message_attachment_menu_component.html.erb

```diff
@@ -4,6 +4,12 @@
       <%= render Icons::ArrowDownTrayComponent.new(css_classes: "h-5 w-5") %>
       Stiahnuť
     <% end %>
+    <% if @message_attachment.xml? %>
```

Comment:
Toto sa da vzdy? Nepotrebujeme tu overovat napr ci mame tu transformaciu?

Reply:
Overujeme to az vnutri v tej metode, nakolko ak XSLT nemame, tak dotiahneme. Nechcela som tym padom vyvolavat stahovanie XSLT uz pri otvoreni spravy, ale az ked bude zaujem o transformaciu.
Ak by sa nahodou nepodarilo transformacia (z akehokolvek dovodu), tak sa zobrazi pouzivatelovi flash message.

Reply:
Imho by sa to xslt malo stiahnut ked stahujeme spravu. Takyto lazy load co sa udeje po kliknuti usera je imho zvlastny.

Reply:
Okay, upravim

Reply:
Upravene v #389